ISSUE_TEMPLATE/testing.md

```diff
@@ -98,13 +105,12 @@ fedora-coreos-stream-generator -releases=https://fcos-builds.s3.amazonaws.com/pr
 A reviewer can validate the `start_epoch` time by running `date -u -d @<EPOCH>`. An example of encoding and decoding in one step: `date -d '2019/09/10 14:30UTC' +%s | xargs -I{} date -u -d @{}`. 
 
 - [ ] Commit the changes and open a PR against the repo.
+- [ ] Post a link to the PR as a comment to this issue
 - [ ] Wait for the PR to be approved.
 - [ ] Once approved, merge it and push the content to S3:
 
 ```
-aws s3 cp --acl public-read --cache-control 'max-age=60' streams/testing.json s3://fcos-builds/streams/testing.json
-
-aws s3 cp --acl public-read --cache-control 'max-age=60' updates/testing.json s3://fcos-builds/updates/testing.json
+aws s3 sync --acl public-read --cache-control 'max-age=60' --exclude '*' --include 'streams/*' --include 'updates/*' . s3://fcos-builds
```

Comment:
This felt safer before when we were targeting a single file. What if someone has some junk files in their repo? I think I'd feel better about leaving ti the way it was.

Reply:
That is unless there is some other improvements here other than collapsing two commands into one.

Reply:
I guess so. I mean, it's two commands right now, but eventually it'd be 6 commands when we add the other two streams. Seemed cleaner to just provide the one-liner.

But, I'm really hoping we get to a place soon where we don't run this command by hand and just have a syncer take care of it. So, meh... I can switch it back if you'd like.